src/marketing/create-new-rec.md

```diff
@@ -4,27 +4,39 @@ tag: product-recommendations
 
 1. Click the **Create Recommendation** button.
 
-1. In the **Name your Recommendation** section, enter a name that uniquely identifies the recommendation. This is used for internal tracking only. Your users will not see this name.
+1. In the **Name your Recommendation** section, enter a name that uniquely identifies the recommendation. This is used for internal tracking only. Your shoppers will not see this name.
+
+1. In the **Storefront display label** section, enter the name for your recommendation, such as "We found other products you might like!". This is the name your shoppers will see on the storefront. Magento provides the [following guidance for label names]({% link marketing/recommendation-best-practices.md %}#bplabels).
 
-1. In the **Select page type** section, select the page on which you want the recommendation to appear.
+1. In the **Select page type** section, select the page on which you want the recommendation to appear. Options are: **Home Page**, **Category**, **Product Detail**, **Cart**, and **Confirmation**.
 
 1. In the **Select Recommendation type** section, specify the type of recommendation you want to appear on the selected page type. Note that certain page types [restrict what type of recommendation]({% link marketing/product-recommendations.md %}#supportedrecs) you can select.
 
-1. In the **Select placement** section, specify the [container where you want the recommendation]({% link marketing/product-recommendations.md %}#productrecplacement) to appear on the page.
+1. In the **Choose number of products** section, use the slider to specify how many products you want to appear in this recommendation unit. The default is `5`. Maximum is `20`.
 
-1. The **Choose position** section displays all recommendations (if any) created for the page type you selected above. You can select and move the rows in the **Choose position** table to order the recommendations within the container.
+1. In the **Select placement** section, specify the [container where you want the recommendation unit]({% link marketing/product-recommendations.md %}#productrecplacement) to appear on the page. Options are **At the bottom of main content** or **At the top of main content**.
 
-1. In the **Choose number of products** section, use the slider to specify how many products you want to appear in this recommendation. The default is `5`. Maximum is `20`.
-
-1. In the **Storefront display label** section, enter the name for your recommendation. This is the name your shoppers will see on the storefront. Magento provides the [following guidance for label names]({% link marketing/recommendation-best-practices.md %}#bplabels).
+1. The **Choose position** section displays all recommendations (if any) created for the page type you selected above. You can select and move the rows in the **Choose position** table to order the recommendations within the container.
```

Comment:
This "step" is more informative than a direction. I would either make it a follow on for the prior step (don't use "above") or rework it differently as an optional step/directive.

```suggestion
   The **Choose position** section displays all recommendations (if any) created for the page type you selected. You can select and move the rows in the **Choose position** table to order the recommendations within the container.
```